### ISSUE TITLE
Error on Load

### ISSUE THREAD

**Issue by MageFX** [ustc-zzzz/VirtualChest#42]
https://pastebin.com/tytZSfDM

MinecraftForge v14.23.1.2555
spongeforge-1.12.2-2555-7.1.0-BETA-2819

**Comment by ustc-zzzz** [comment]
```
Caused by: java.lang.IllegalArgumentException: Multiple entries with same key: 
    Cooldown=SpongeKey{name=Cooldown, id=sponge:cooldown, valueToken=org.spongepowered.api.data.value.mutable.Value<java.lang.Integer>, elementToken=java.lang.Integer, query=Cooldown} and 
    Cooldown=SpongeKey{name=Cooldown, id=entity-commands:cooldown, valueToken=org.spongepowered.api.data.value.mutable.Value<java.lang.String>, elementToken=java.lang.String, query=Cooldown}
```
Seems that there are two keys with duplicate queries in key registry. One is provided by `sponge` and another is provided by `entity-commands`.
I had searched both ore and sponge forum and did not find any plugin related to both `entity` and `command`. Could you please provide a list of plugins you are using in your server?

**Comment by randombyte-developer** [comment]
I'm the author of entity-commands. The IDs really are the identification for the data values. Try to change your key map from <Query, Key> to <String, Key> where the string is the key ID.

**Comment by PixelmonPRO** [comment]
I have this error too

**Comment by ustc-zzzz** [comment]
Fixed in [v0.4.6](https://github.com/ustc-zzzz/VirtualChest/releases/tag/v0.4.6)
